src/components/Dashboard/TodoList.vue

```diff
@@ -1176,9 +1191,12 @@ export default class TodoList extends Vue {
       const paymentStatusCode = header.paymentStatusCode
       const payErrorObj = paymentStatusCode && await PayServices.getPayErrorObj(paymentStatusCode)
 
-      const draft = changeOfRegistration
       // FUTURE: verify that this checks for all possible data:
-      const haveData = Boolean(draft?.offices || draft?.contactPoint || draft?.parties)
+      const haveData = Boolean(
+        changeOfRegistration.offices ||
+        changeOfRegistration.contactPoint ||
+        changeOfRegistration.parties
+      )
```

Comment:
No need for optional chaining since `changeOfRegistration` was already checked on line 1178.

Ditto with `conversion` below.

Reply:
Outdated conversation.